src/udf/Cargo.toml

```diff
@@ -26,6 +31,8 @@ tokio = { version = "0.2", package = "madsim-tokio", features = [
 ] }
 tonic = { workspace = true }
 tracing = "0.1"
+wasmtime = { version = "10", features = ["component-model"] }
+wasmtime-wasi = { version = "10" }
```

Comment:
I'm trying to update it to the latest v15. But there are a lot of API changes...

Reply:
Done in f65ed64. The Rust function works, but the golang side is left unfixed.